main.go

```diff
@@ -46,5 +50,12 @@ func main() {
 	// Register template functions.
 	beego.AddFuncMap("i18n", i18n.Tr)
 
+	// serve securely if the certificates are present
+	if _, err := os.Open(ServingCertFile); err == nil {
```

Comment:
Since we don't care about opening the file, could we replace this with `os.Stat`?

Also we're checking only one of the files, I see that beego we'll fail to start if one of those files cannot be opened:

```console
2016/03/02 13:07:25 [app.go:32] [D] Loading language: en-US 
2016/03/02 13:07:25 [main.go:30] [I] Web IM 0.1.1.0227 
2016/03/02 13:07:25 [asm_amd64.s:2232] [I] http server Running on :8888
2016/03/02 13:07:25 [asm_amd64.s:2232] [I] https server Running on :10443
2016/03/02 13:07:25 [asm_amd64.s:2232] [C] ListenAndServeTLS: %!(EXTRA *os.PathError=open /foo/bar: no such file or directory)
```